front/src/modules/trainschedule/components/ImportTrainSchedule/ImportTrainScheduleTrainsList.tsx

```diff
@@ -77,36 +88,56 @@ const ImportTrainScheduleTrainsList = ({
 
   const [postTrainSchedule] =
     osrdEditoastApi.endpoints.postTimetableByIdTrainSchedules.useMutation();
+  const [postPacedTrain] = osrdEditoastApi.endpoints.postTimetableByIdPacedTrains.useMutation();
 
   const dispatch = useAppDispatch();
 
   async function generateTrainSchedules() {
     try {
-      let payloads;
+      let trainSchedulePayloads: TrainScheduleBase[] = [];
+      let pacedTrainPayloads: PacedTrain[] = [];
 
       if (trainsXmlData.length > 0) {
-        payloads = generateTrainSchedulesPayloads(trainsXmlData, true);
-      } else if (trainsJsonData.length > 0) {
-        payloads = trainsJsonData;
+        trainSchedulePayloads = generateTrainSchedulesPayloads(trainsXmlData, true);
+      } else if (trainSchedulesJsonData.length > 0 || pacedTrainsJsonData.length > 0) {
+        trainSchedulePayloads = trainSchedulesJsonData;
+        pacedTrainPayloads = pacedTrainsJsonData;
       } else {
```

Comment:
We could put some docs here : 
```suggestion
      // Viriato import (TODO Paced train : handle viriato imports for paced trains)
      if (trainsXmlData.length > 0) {
        trainSchedulePayloads = generateTrainSchedulesPayloads(trainsXmlData, true);
        // Json import
      } else if (trainSchedulesJsonData.length > 0 || pacedTrainsJsonData.length > 0) {
        trainSchedulePayloads = trainSchedulesJsonData;
        pacedTrainPayloads = pacedTrainsJsonData;
        // Open data import (only handle trainSchedules)
      } else {
        trainSchedulePayloads = generateTrainSchedulesPayloads(formattedTrainsList, false);
      }
```

Reply:
Done